app/src/main/java/org/simple/clinic/bp/assignbppassport/ScannedQrCodeUiRenderer.kt

```diff
@@ -0,0 +1,14 @@
+package org.simple.clinic.bp.assignbppassport
+
+import org.simple.clinic.mobius.ViewRenderer
+import org.simple.clinic.patient.businessid.Identifier
+
+class ScannedQrCodeUiRenderer(private val ui: ScannedQrCodeUi) :ViewRenderer<BpPassportModel>{
+  override fun render(model: BpPassportModel) {
+    if (model.identifier.type == Identifier.IdentifierType.IndiaNationalHealthId){
```

Comment:
Can we import the package before `IndiaNationalHealthId`. This would help in better readability.